server/app/services/triplers.js

```diff
@@ -327,6 +280,11 @@ async function searchTriplersAmbassador(req) {
 // searching as admin removes constraint of requiring no claims relationship
 // as well as removing constraint of requiring no upgraded status
 async function searchTriplersAdmin(req) {
```

Comment:
Wishful thinking: Can this endpoint go away? It seems like if we need special admin search powers we could build it into the regular search method above.